Index: app/src/main/java/com/lampa/emotionrecognition/UserActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/lampa/emotionrecognition/UserActivity.java b/app/src/main/java/com/lampa/emotionrecognition/ListenerActivity.java
rename from app/src/main/java/com/lampa/emotionrecognition/UserActivity.java
rename to app/src/main/java/com/lampa/emotionrecognition/ListenerActivity.java
--- a/app/src/main/java/com/lampa/emotionrecognition/UserActivity.java	(revision bdb8df5aedd9a3727cb8dfb325e7911aaf9054c3)
+++ b/app/src/main/java/com/lampa/emotionrecognition/ListenerActivity.java	(date 1639593435243)
@@ -1,7 +1,7 @@
 package com.lampa.emotionrecognition;
 
 //
-public class UserActivity extends UserClass{
+public class ListenerActivity extends UserClass{
 
     public void watchHistoryOnClick(){
 
